src/js/letters/utils/helpers.js

```diff
@@ -58,21 +37,165 @@ export const letterContent = {
-    hasSpecialMonthlyCompensation: <div>You <strong>are</strong> service-connected for loss of or loss of use of a limb, or you are totally blind in or missing at least one eye</div>,
-    hasSurvivorsIndemnityCompensationAward: <div></div>,
-    hasSurvivorsPensionAward: <div></div>,
-    monthlyAwardAmount: <div>Your current monthly award amount is <strong>${currentValue}</strong></div>,
-    serviceConnectedPercentage: <div>Your combined service-connected evaluation is <strong>{currentValue}%</strong></div>,
-  };
+// Options returned by the benefit summary letter request that should be offered in
+// the checkbox list regardless of their values (e.g., true, false, 'unavailable', or other)
+// All other options are conditionall displayed, depending on the value
+export const optionsToAlwaysDisplay = [
+  'hasChapter35Eligibility',
+  'hasDeathResultOfDisability',
+  'hasServiceConnectedDisabilities',
+  'hasSurvivorsIndemnityCompensationAward',
+  'hasSurvivorsPensionAward',
+  'serviceConnectedPercentage'
+];
+
+const benefitOptionText = {
+  hasNonServiceConnectedPension: {
+    'true': {
+      veteran: <div>Your non-service connected pension information</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: undefined,
+      dependent: undefined
+    }
+  },
+  hasServiceConnectedDisabilities: {
+    'true': {
+      veteran: <div>You have one or more service-connected disabilities</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: <div>You <strong>do not</strong> have one or more service-connected disabilities</div>,
+      dependent: undefined
+    }
+  },
+  hasSurvivorsIndemnityCompensationAward: {
+    'true': {
+      veteran: undefined,
+      dependent: <div>You <strong>are</strong> receiving Dependency and Indemnity Compensation</div>
+    },
+    'false': {
+      veteran: undefined,
+      dependent: <div>You <strong>are not</strong> receiving Dependency and Indemnity Compensation</div>
+    }
+  },
+  hasSurvivorsPensionAward: {
+    'true': {
+      veteran: undefined,
+      dependent: <div>You <strong>are</strong> receiving Survivors Pension></div>
+    },
+    'false': {
+      veteran: undefined,
+      dependent: <div>You <strong>are not</strong> receiving Survivors Pension></div>
+    }
+  },
+  hasAdaptedHousing: {
+    'true': {
+      veteran: <div>You <strong>have</strong> been found entitled to a Specially Adapted Housing (SAH) and/or Special Home Adaptation (SHA) grant</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: undefined,
+      dependent: undefined
+    }
+  },
+  hasChapter35Eligibility: {
+    'true': {
+      veteran: <div>You are considered to be totally and permanently disabled solely due to your service-connected disabilities</div>,
+      dependent: <div>You <strong>are</strong> eligible for Dependents' Educational Assistance (Chapter 35).</div>
+    },
+    'false': {
+      // This doesn't look right: getting clarification in
```

Comment:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3685 has some updates (bolding and language), but still has one detail to be resolved.